### ISSUE TITLE
currency symbol in wrong location

### ISSUE THREAD

**Issue by smmmadden** [Zrips/Jobs#213]
**/jobs info Enchanter** returns this for Spigot 1.13 are incorrectly showing the position of the currency symbol at the end of the amount, but the $ (USA dollar) always goes in front. (e.g. $3.38)
git-Spigot-2b0e71c-c0df4b8 (MC: 1.13) (Implementing API version 1.13-R0.1-SNAPSHOT)

![image](https://user-images.githubusercontent.com/10309346/44005299-4fbcd5ba-9e3f-11e8-9a7d-d79414116ac7.png)

**Comment by montlikadani** [comment]
You will change this in the message file.

**Comment by smmmadden** [comment]
Thanks, that did it.  please update the default for messages_en.yml so it reads like this:
      money: ' &2$ %money%'
Results look much better like this:
![image](https://user-images.githubusercontent.com/10309346/44010377-00a354f4-9e80-11e8-8c5a-46fb023377ef.png)
I'm sure some of the others have the currency symbol on the left as well.

**Comment by smmmadden** [comment]
I disagree - this is not a minecraft language issue, it's a Jobs configuration issue not properly formatting currencies based on the language files.  @Zrips what do you think?  It looks like it was just configured all the same to match points and experience labels being at the end and a simple mistake that should be corrected in the default file.

**Comment by montlikadani** [comment]
@smmmadden 
What I said is not a mistake, that's the way it is. Please do **not** continue the effort here because you can do this yourself in the messages file.
